gpt_connection.py

```diff
@@ -0,0 +1,29 @@
+
+
+from openai import OpenAI
+import os
+from dotenv import load_dotenv
+from models import Experience, Education
+load_dotenv()
+
+client = OpenAI(
+    api_key=os.getenv("OPENAI_API_KEY")
+)
+
+def get_improvement(model : Experience | Education):
+    prompt = f"Improve the following description for the {model.__class__.__name__}: {model.description}"
+
+    response = client.chat.completions.create(
+        model="gpt-4o-mini",
+        messages=[
+            {"role": "system", "content": "You are a helpful assistant that improves descriptions for resumes."},
+            {"role": "assistant", "content": "Here is the improved description:"},
+            {"role": "system", "content": f"Consider the following information about the {model.__class__.__name__}: {model.model_dump_json()}"},
+            {"role": "user", "content": prompt},
+        ]
+    )
+
+    return response.choices[0].message.content
```

Comment:
What is the response is None here or does not have the full information? Let's have some error handling to make sure that we return either the correct information or some error